extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -873,12 +875,36 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  /** @override **/
+  attemptChangeSize(newHeight, newWidth) {
+    // Store original size of slot in order to allow re-expansion on
+    // unlayoutCallback so that it is reverted to original size in case
+    // of resumeCallback.
+    this.originalSlotSize_ = this.getIntersectionElementLayoutBox();
```

Comment:
`#getLayoutBox`. `#getIntersectionElementLayoutBox` is meant to be overridable for getting the iframe's layout box.